src/monodroid/monodroid.targets

```diff
@@ -15,13 +15,7 @@
     <_EmbeddedBlobDestination Include="jni\machine.config.include" />
   </ItemGroup>
   <Target Name="_BuildRuntimes"
-      DependsOnTargets="_GenerateIncludeFiles;_BuildAndroidRuntimes;_BuildHostRuntimes">
-  </Target>
-  <Target Name="_GenerateIncludeFiles"
```

Comment:
Why are you removing this target?

Removing `support/nl.c` does not in any way change our need for `monodroid_config` or `monodroid_machine_config`, both of which are generated by `<GenerateMonoDroidIncludes/>`; see also:

https://github.com/xamarin/xamarin-android/blob/60c1ee42437066366b4da5f94ad47d41dcb26284/src/monodroid/jni/monodroid-glue.cc#L2009-L2010

DO NOT remove the `_GenerateIncludeFiles` target.  You only need to update the `Inputs` so that `jni\config.h` isn't specified.

***Or...***, leave this entirely unchanged, and "revert" the `config.h` removal (and just remove the declarations of `CreateNLSocket()`/etc.), so that the `#include <config.h>` usage continues to work.

Reply:
Ah okay my bad, I misread what you'd said on Slack. Leaving what's just basically an empty config file seems gross, but I guess that's the easiest option for now...